firmware/main/conductor/conductor.ino

```diff
@@ -64,10 +307,56 @@ void setup() {
   staff.setDebugVerbose(DEBUG_VERBOSE);
 
   playbackTimer.begin(playColumn, PLAYBACK_PERIOD_DEFAULT);
+
+  Serial.begin(9600);
+  AudioMemory(40);
+  sgtl5000_1.enable();
+  sgtl5000_1.volume(.8);
+  SPI.setMOSI(SDCARD_MOSI_PIN);
+  SPI.setSCK(SDCARD_SCK_PIN);
+  if (!(SD.begin(SDCARD_CS_PIN))) {
+    while (1) {
+      Serial.println("Unable to access the SD card");
+      delay(500);
+    }
+  }
```

Comment:
This block only tries the SD card once, then hangs in an infinite while loop. When installed the Teensy should just keep trying to access the SD card. That way it could *maybe* recover and start working again.

Also could consider a software reset if any errors are detected, so as to re-initialize the system.